docs/en/advanced-features/manual-apis/toolkit-trace.md

```diff
@@ -60,9 +60,9 @@ trace.StopSpan()
 
 ### Add Span’s Tag and Log
 
-Use `trace.SetLog` to record log in span.
+Use `trace.SetLog()` to record log in span.
 
-Use `trace.SetTag` to add tag to span, the parameters of tag are two String which are key and value respectively.
+Use `trace.SetTag()` to add tag to span, the parameters of tag are two String which are key and value respectively.
```

Comment:
Thank you very much.
And  the method that implements the relevant functionality is named "AddLog," but there is an error where it is written as "SetLog." Could you please correct it?

Reply:
Indeed, just changed it!